src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,10 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Ball[] luckyBalls =
```

Comment:
Create an instance of a Lottery class and in the loop use method `getRandomBall()`. 

Don't call method `toString()` in `System.out.println()`, cause this method calls `toString()` as default.